presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -480,13 +478,15 @@ void PrestoServer::initializeVeloxMemory() {
   auto systemConfig = SystemConfig::instance();
   uint64_t memoryGb = nodeConfig->nodeMemoryGb(
       [&]() { return systemConfig->systemMemoryGb(); });
-  LOG(INFO) << "Starting with node memory " << memoryGb << "GB";
+  LOG(INFO) << "STARTUP: Starting with node memory " << memoryGb << "GB";
   std::unique_ptr<cache::SsdCache> ssd;
   const auto asyncCacheSsdGb = systemConfig->asyncCacheSsdGb();
   if (asyncCacheSsdGb) {
     constexpr int32_t kNumSsdShards = 16;
     cacheExecutor_ =
         std::make_unique<folly::IOThreadPoolExecutor>(kNumSsdShards);
+    LOG(INFO) << "STARTUP: Initializing SSD cache with " << asyncCacheSsdGb
+              << "GB";
```

Comment:
Should we add more info int o log here?
Like systemConfig->asyncCacheSsdCheckpointGb() and systemConfig->asyncCacheSsdDisableFileCow(), since we are logging.

Reply:
We surely can